index.html

```diff
@@ -1,249 +1,338 @@
+                    class="social-img"
+                    alt="medium"
+                  />
+                </a>
+              </li>
+              <li class="social-item">
+                <a href="#">
+                  <img
+                    src="assets/github.png"
+                    class="social-img"
+                    alt="github"
+                  />
+                </a>
+              </li>
+              <li class="social-item">
+                <a href="#">
+                  <img
+                    src="assets/finger.png"
+                    class="social-img"
+                    alt="finger"
+                  />
+                </a>
+              </li>
+            </ul>
+            <button class="btn-project">Get my resume</button>
+          </div>
+          <div class="skillsets">
+            <div class="skillset">
+              <div class="language-title-container">
+                <h4>Languages</h4>
+                <span><i class="bx bx-chevron-down"></i></span>
+              </div>
+              <ul class="languages-list">
+                <li class="language">
+                  <img src="assets/javascript-logo.png" alt="javascript" />
+                  <span>JavaScript</span>
+                </li>
+                <li class="language">
+                  <img src="assets/html5-logo.png" alt="html" />
+                  <span>HTML</span>
+                </li>
+                <li class="language">
+                  <img src="assets/css-logo.png" alt="css" />
+                  <span>CSS</span>
+                </li>
+              </ul>
+            </div>
+            <div class="skillset">
+              <div class="language-title-container">
+                <h4>Frameworks</h4>
+                <span><i class="bx bx-chevron-right"></i></span>
+              </div>
+            </div>
+            <div class="skillset">
+              <div class="language-title-container">
+                <h4>Skills</h4>
+                <span><i class="bx bx-chevron-right"></i></span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="contact-section">
+        <div class="contact-form-section">
+          <div class="contact-form">
+            <h4 class="contact-heading">Contact me</h4>
+            <p class="contact-text">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/mwkabeev" method="POST">
+              <fieldset class="fieldset">
+                <div class="field">
+                  <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    maxlength="30"
+                    class="control"
+                    value="Olawale"
```

Comment:
- [ ] **OPTIONAL** I suggest deleting the `value` attribute and just leave the `placeholder`